pkg/third-party/event-calendar/index.html

```diff
@@ -0,0 +1,50 @@
+<html>
```

Comment:
is this like a smoke test, or is it part of the third party library?
(i'm asking because i'm often looking for them by `smoke` name, and it improves discoverability :))

Reply:
It's a smoke test that came with the library, we can change the name.

Reply:
Also, fwiw, this is the least important calendar. I think `@event-calendar` is superior.